client/app/scripts/charts/edge-container.js

```diff
@@ -50,14 +56,9 @@ class EdgeContainer extends React.Component {
   }
 
   render() {
-    const { layoutPrecision, points } = this.props;
+    const { layoutPrecision } = this.props;
     const other = _.omit(this.props, 'points');
 
-    if (layoutPrecision === 0) {
```

Comment:
But if we don't want to animate the edges on graphs with `>= 50` nodes, then we are very unlikely to ever cross the old limit of `30` waypoints used with `Motion`. I thought the whole point of this story was to lift this limit, so that the bigger graphs could also animate properly. :)

 `pointsMap` will only be used on small graphs, so it would be very unlikely that we'd cross the limit of `30` waypoints there.